### DISCUSSION TITLE
How to go about using slug as nuxt-link to an article?

### DISCUSSION THREAD

**Discussion by marvthedev** [strapi/strapi#7687] [Legacy Topics]
How can I use slug instead of id as a url to my single article pages?

I have on my pages/articles/index.vue:
 ```
<nuxt-link :to="{ name: 'articles-slug', params: { slug: article.Slug }}" tag="a">See Article
                  </nuxt-link>
```

On my pages/articles/_slug.vue
```
export default {
  data() {
    return {
      articles: {},
      api_url: process.env.strapiBaseUri,
    };
  },
  apollo: {
    articles: {
      prefetch: true,
      query: singleArticle,
      variables() {
        return { slug: this.$route.params.slug };
      }
    }
  }
};
</script>
```

I keep getting an error: Network error: Response not successful: Received status code 400

**Comment by derrickmehaffy** [comment]
Let me ping @Mcastres and @remi2j to see if they can help, normally @mlmarinas we don't usually have the ability to help with outside frameworks here in Github discussions.
